Automattic/pocket-casts-android
# Convert "Customise media actions" to Compose

## Description

This change converts the "Customise media actions" page to Compose. 

## Testing Instructions

- Play an episode
- Tap on the Profile tab
- Tap on the settings cog
- Tap on General
- Tap on Media notification controls
- ✅ Pull down the notification shade and check the actions match (you might only be able to see two)
- On the "Media notification controls" page drag to rearrange the actions
- ✅ Pull down the notification shade and check the actions match
- Tap on the "Show custom media actions" to disable the actions
- ✅ Verify you can drag the actions and check the notification actions are gone 

## Screenshots

| Before | After |
| --- | --- |
| ![Screenshot_20240906_214254](https://github.com/user-attachments/assets/e3ce0b62-bb02-439a-8c9a-0b4cbf8abefd) | ![Screenshot_20240906_220602](https://github.com/user-attachments/assets/f56d3461-b7e9-4366-be92-845c180ca58d) | 
| ![Screenshot_20240906_214310](https://github.com/user-attachments/assets/70db2897-1f9a-4e62-b565-f59a1626950d) | ![Screenshot_20240906_220616](https://github.com/user-attachments/assets/376a621a-54b4-4429-afd8-c113f8bcefc9) |

https://github.com/user-attachments/assets/c77cd70d-0cee-4f37-a13f-f27fef322ccf

## Checklist
 
#### I have tested any UI changes...
